# HG changeset patch
# User Ken Brown <kbrown@cornell.edu>
# Date 1425572886 18000
#      Thu Mar 05 11:28:06 2015 -0500
# Node ID 3156692cdd6e5f8d535c730dd499ddf750c90621
# Parent  3e0a47cc519e131f658131387a5dd118cacff85d
* src/m4/signal.m4: Remove obsolete Cygwin-specific code

diff -r 3e0a47cc519e -r 3156692cdd6e src/m4/signal.m4
--- a/src/m4/signal.m4	Thu Mar 05 10:48:49 2015 -0500
+++ b/src/m4/signal.m4	Thu Mar 05 11:28:06 2015 -0500
@@ -20,11 +20,6 @@
 #include <unistd.h>
 #endif
 #include <signal.h>
-#ifdef __CYGWIN32__
-/* On Cygwin32 version 18, this test would hang (SIGALRM not being signalled).
- * Let it fail instead. */
-#error "better fail than hang"
-#endif
 volatile int gotsig=0;
 void sigalrm_handler() { gotsig=1; }
 int got_sig () { return gotsig; }
@@ -55,11 +50,6 @@
 #include <unistd.h>
 #endif
 #include <signal.h>
-#ifdef __CYGWIN32__
-/* On Cygwin32 version 18, this test would hang (SIGALRM not being signalled).
- * Let it fail instead. */
-#error "better fail than hang"
-#endif
 volatile int gotsig=0;
 volatile int wasblocked=0;
 typedef void (*signal_handler_t) (int);
@@ -101,11 +91,6 @@
 #include <unistd.h>
 #endif
 #include <signal.h>
-#ifdef __CYGWIN32__
-/* On Cygwin32 version 18, this test would hang (SIGALRM not being signalled).
- * Let it fail instead. */
-#error "better fail than hang"
-#endif
 volatile int gotsig=0;
 volatile int somewereblocked=0;
 typedef void (*signal_handler_t) (int);
@@ -156,11 +141,6 @@
 #include <unistd.h>
 #endif
 #include <signal.h>
-#ifdef __CYGWIN32__
-/* On Cygwin32 version 18, this test would hang (SIGALRM not being signalled).
- * Let it fail instead. */
-#error "better fail than hang"
-#endif
 typedef void (*signal_handler_t) (int);
 signal_handler_t mysignal (int sig, signal_handler_t handler)
 { struct sigaction old_sa;
@@ -202,11 +182,6 @@
 #include <unistd.h>
 #endif
 #include <signal.h>
-#ifdef __CYGWIN32__
-/* On Cygwin32 version 18, this test would hang (SIGALRM not being signalled).
- * Let it fail instead. */
-#error "better fail than hang"
-#endif
 typedef void (*signal_handler_t) (int);
 signal_handler_t mysignal (int sig, signal_handler_t handler)
 { struct sigaction old_sa;
